pages/index.tsx

```diff
@@ -6,79 +6,22 @@ const Home: NextPage = () => {
   return (
     <div className="flex min-h-screen flex-col items-center justify-center py-2">
       <Head>
-        <title>Create Next App</title>
-        <link rel="icon" href="/favicon.ico" />
+        <title>Defaang</title>
       </Head>
 
-      <main className="flex w-full flex-1 flex-col items-center justify-center px-20 text-center">
-        <h1 className="text-6xl font-bold">
-          Welcome to{' '}
-          <a className="text-blue-600" href="https://nextjs.org">
-            Next.js!
-          </a>
+      <main className="w-full  px-20 text-center">
+        <h1 className="text-5xl font-medium">
+        Hello There!  Welcome to Defaang. <br />
```

Comment:
Can we change Defaang to defaang? I think the lowercased brand is going to be a bit more friendly

Reply:
Okay